.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: false
 language: node_js
 
 node_js:
-  - "6.9.3"
+  - node
```

Comment:
Yeah, that's a weird artifact of `node` and `iojs` being options.
Would assume that means `7.x` today, so `6` should still be in the list.